src/txmempool.cpp

```diff
@@ -356,7 +356,7 @@ void CTxMemPool::AddTransactionsUpdated(unsigned int n)
 void CTxMemPool::addUnchecked(const CTxMemPoolEntry &entry, setEntries &setAncestors, bool validFeeEstimate)
 {
     // Add to memory pool without checking anything.
-    // Used by AcceptToMemoryPool(), which DOES do
+    // Used by AcceptToMemoryPool(::ChainstateActive(), ), which DOES do
```

Comment:
I think the scripted-diff is too much expansive here.

EDIT: note for reviewers, fixed in next commit.

Reply:
Yeah I should have used a clang-tidy script for this... Oh well, perhaps we can set that up in the future.